backbone/src/main/java/org/researchstack/backbone/utils/DateUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.researchstack.backbone.utils;
```

Comment:
Is this used at all?

Reply:
I didn't do the optimize import for the moment. I will do it before merging :)